### PR TITLE
[RIO/2022] new speaker and sponsors

### PR DESCRIPTION
**Pull request by camilla-m**
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*

If you are adding or removing organisers, please email info@devopsdays.org with the details of who is being added and removed, along with the full names and email addresses of the new team, so we can update Slack and the mailing list.

### CODE REVIEW

File: content/events/2022-rio-de-janeiro/speakers/danielle-monteiro.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+linktitle = "danielle-monteiro"
+
++++
+
+Sou mestra em Engenharia da Computação, TEDx Speaker, autora do blog e do canal PrimeirosDados.com
+
+Minha carreira na área de dados tem mais de 20 anos. Já fui DBA, arquiteta, consultora, engenheira de dados e agora sou Cloud Solutions Architect de Dados e Inteligência Artificial na Microsoft.
+
+Durante a minha carreira ganhei diversos prêmios dentro e fora do Brasil, como por exemplo:
+– MongoDB Champion;
```
Comment (glasnt) [inline content/events/2022-rio-de-janeiro/speakers/danielle-monteiro.md:16]:
```suggestion

– MongoDB Champion;
```

Without a space, this won't appear as a list in markdown. Unsure if intended, though.

**Comment by glasnt** [comment]
There are also no sponsor changes in this PR, unsure if they're pending

File: content/events/2022-rio-de-janeiro/speakers/danielle-monteiro.md (hunk 1 above)
Comment (camistoryblok) [reply content/events/2022-rio-de-janeiro/speakers/danielle-monteiro.md:16]: nice, thank you!

**Comment by camistoryblok** [comment]
> There are also no sponsor changes in this PR, unsure if they're pending

changes are here https://github.com/devopsdays/devopsdays-web/blob/84e372d639d922e8acd312051037d3aabf44d2f6/data/events/2022-rio-de-janeiro.yml

**Pull request merged**